docs/latest/design/roadmap.md

```diff
@@ -34,11 +34,17 @@ contributing to the project.
 
 ### [v0.3.0][v0.3.0]: Drive Advanced Features through Extension Mechanisms
 
-- Global Rate Limiting
-- AuthN/AuthZ- [Issue #336][336].
-- Lets Encrypt Integration
+- Support extended Gateway API fields [Issue #707][707].
+- Support experimental Gateway APIs such as TCPRoute [Issue #643][643], UDPRoute [Issue #641][641] and GRPCRoute [Issue #642][642].
+- Establish guidelines for leveragaing Gateway API extensions [Issue #675][675].
+- Rate Limiting [Issue #670][670].
+- Authentication [Issue #336][336].
```

Comment:
IMO we're still too aggressive with our v0.3 deliverables. For example, #642 should not be on the roadmap until someone is assigned and has confirmed delivering the feature for the milestone.